### PR TITLE
Solutions: Add Flare Systems Firework connector

### PR DESCRIPTION
**Pull request by aviau**
Hello!

This PR adds a connector for Flare System's Firework.

Please let me know if there is anything that I need to change.

Cheers,

### CODE REVIEW

File: Logos/FlareSystems.svg [hunk 1]
```diff
@@ -0,0 +1,105 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 220.553 220.991">
```
Comment (elforb) [inline Logos/FlareSystems.svg:1]: While this SVG renders properly, it fails validation on our internal tools, potentially due to the extremely long path objects.

File: DataConnectors/Flare Systems Firework/Connector_REST_API_FlareSystemsFirework.json [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+                    "description": "At this point, you may configure alerts to be sent to Azure Sentinel the same way that you would configure regular email alerts."
+                },
+                {
+                    "title": "",
+                    "description": "For a more detailed guide, refer to the Firework documentation."
+                }
+            ]
+        }
+    ]
+}
```
Comment (elforb) [inline DataConnectors/Flare Systems Firework/Connector_REST_API_FlareSystemsFirework.json:97]: Please add a Metadata object within this file. See https://github.com/Azure/Azure-Sentinel/blob/master/DataConnectors/Templates/Connector_REST_API_template.json for further detail.

**Comment by elforb** [comment]
Please also move the data connector and logo files into the solutions folder.
- `/Solutions/Flare Systems Firework/Data Connectors/Connector_REST_API_FlareSystemsFirework.json`
- `/Solutions/Flare Systems Firework/Data Connectors/Logo/FlareSystems.svg`

Sample data can remain in its current location.

File: DataConnectors/Flare Systems Firework/Connector_REST_API_FlareSystemsFirework.json [hunk 3]
```diff
@@ -0,0 +1,97 @@
+{
```
Comment (elforb) [inline DataConnectors/Flare Systems Firework/Connector_REST_API_FlareSystemsFirework.json:1]: Please also move the data connector and logo files into the solutions folder.

/Solutions/Flare Systems Firework/Data Connectors/Connector_REST_API_FlareSystemsFirework.json
/Solutions/Flare Systems Firework/Data Connectors/Logo/FlareSystems.svg
Sample data can remain in its current location.

**Review by elforb — CHANGES_REQUESTED** [review]

File: DataConnectors/Flare Systems Firework/Connector_REST_API_FlareSystemsFirework.json (hunk 3 above)
Comment (aviau) [reply DataConnectors/Flare Systems Firework/Connector_REST_API_FlareSystemsFirework.json]: Done!

File: DataConnectors/Flare Systems Firework/Connector_REST_API_FlareSystemsFirework.json (hunk 2 above)
Comment (aviau) [reply DataConnectors/Flare Systems Firework/Connector_REST_API_FlareSystemsFirework.json]: Done! I wasn't sure what to set for the `id`, let me know if I need to change it.

File: Logos/FlareSystems.svg (hunk 1 above)
Comment (aviau) [reply Logos/FlareSystems.svg]: Done!

File: Solutions/Flare Systems Firework/Data Connectors/Connector_REST_API_FlareSystemsFirework.json [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+                },
+                {
+                    "title": "",
+                    "description": "For a more detailed guide, refer to the Firework documentation."
+                }
+            ]
+        }
+    ],
+    "metadata": {
+        "id": "FlareSystemsFirework",
```
Comment (elforb) [inline Solutions/Flare Systems Firework/Data Connectors/Connector_REST_API_FlareSystemsFirework.json:98]: This `id` here should be a GUID, as given in [the template property](https://github.com/Azure/Azure-Sentinel/blob/master/DataConnectors/Templates/Connector_REST_API_template.json#L94).

For example: `c3f2c642-54a5-49b4-b135-e05506720765` could be used, as I've verified it does not exist in the repository.

**Review by elforb — CHANGES_REQUESTED** [review]

File: Solutions/Flare Systems Firework/Data Connectors/Connector_REST_API_FlareSystemsFirework.json (hunk 4 above)
Comment (aviau) [reply Solutions/Flare Systems Firework/Data Connectors/Connector_REST_API_FlareSystemsFirework.json]: Ah, thanks. Done!

**Review by v-jayakal — APPROVED** [review]

**Comment by aviau** [comment]
Thank you for the reviews!

**Pull request merged**